tracing-subscriber/src/fmt/mod.rs

```diff
@@ -94,6 +94,29 @@
 //! // Note this will only fail if you try to set the global default
 //! // subscriber multiple times
 //! ```
+//!
+//! ### Composing Layers
+//!
+//! Composing an EnvFilter Layer and a format Layer:
+//!
+//! ```rust
+//! use tracing_subscriber::{fmt, layer::Layer as _, registry::Registry, EnvFilter};
+//!
+//! let default_filter = concat!(module_path!(), "=info");
```

Comment:
if we're going to suggest this behavior, can we explain what it's doing and why we would want to do this? i have personally never written this code & i'm not convinced its worth having in the example...if we wanted to demonstrate this, we could have a separate example showing "if you want to include only traces from your crate, you can do this..." in the filtering docs?